client/src/locales/en-US.json

```
@@ -410,22 +410,22 @@
         "copySelectFolderTitle": "Copy one item | Copy {count} items",
         "copySelectFolderSubtitle": "Current location: {location}",
         "moveSelectFolderTitle": "Move one item | Move {count} items",
-        "moveSuccess": "The element have been moved the choosen folder. | All the elements have been moved to the chosen folder.",
-        "copySuccess": "The element have been copied to the choosen folder. | All the elements have been copied to the chosen folder.",
+        "moveSuccess": "The element have been moved the chosen folder. | All the elements have been moved to the chosen folder.",
+        "copySuccess": "The element have been copied to the chosen folder. | All the elements have been copied to the chosen folder.",
         "open": {
-            "fileFailed": "Could not open file `{name}`",
-            "folderFailed": "Could not open folder `{name}`",
+            "fileFailed": "Failed to open file `{name}`",
+            "folderFailed": "Failed to open folder `{name}`",
             "unavailableOnWeb": "This feature is not yet available in web mode."
         },
-        "linkNotCopiedToClipboard": "Could not copy the link. Check that your browser or device allows copy/paste.",
+        "linkNotCopiedToClipboard": "Failed to copy the link. Check that your browser or device allows copy/paste.",
         "linkCopiedToClipboard": "The link has been copied to the clipboard.",
-        "getLinkError": "Could not get a link to the file",
+        "getLinkError": "Failed to get a link to the file",
         "ImportFile": {
             "title": "Importation",
             "waiting": "waiting",
             "cancelled": "cancelled",
             "failed": "failed",
-            "failedDetails": "Please check you internet connection.",
+            "failedDetails": "Please check your internet connection.",
```

To be consistent with previous messages:
```suggestion
            "failedDetails": "Please make sure you are connected to Internet.",
```